Docs: Update docs for enterprise reporting (#37314)
grafana/grafana: docs/sources/enterprise/reporting.md

A background job runs every 10 minutes and removes temporary CSV files. You can configure how long a CSV file should be stored before being removed by configuring the [temp-data-lifetime]({{< relref "../administration/configuration/#temp-data-lifetime" >}}) setting. This setting also affects how long a renderer PNG file should be stored.

### Scheduling
> Note: Scheduler has been significantly changed in Grafana Enterprise v8.1.

> Scheduled reports can be sent once or repeated on an hourly, daily, weekly and monthly basis, or at custom intervals.  You can also disable scheduling by selecting **Never** if you want to send the report via the API.

Reviewer: ```suggestion
Scheduled reports can be sent once or repeated on an hourly, daily, weekly and monthly basis, or at custom intervals.  You can also disable scheduling by selecting **Never** or if you want to send the report via the API.
```

Author: Not sure about this. I meant to say that "Never" can be used to disable scheduling, for example, when report distribution needs to be triggered via API.

Reviewer: What I meant is that in the current state it reads as if you'd only set schedule to Never to send reports via API, but sometimes you'd do that just not to send a report. I might have misunderstood the sentence though.

Author: You're right. Thank you!

> Scheduled reports can be sent once or repeated on an hourly, daily, weekly and monthly basis, or at custom intervals. You can also disable scheduling by selecting **Never**: for example, if you want to send the report via the API.

Reviewer 2: ```suggestion
Scheduled reports can be sent once or repeatedly on an hourly, daily, weekly, or monthly basis, or at custom intervals. You can also disable scheduling by selecting **Never**: for example, if you want to send the report via the API.
```

{{< figure src="/static/img/docs/enterprise/reports_scheduler_8.1.png" max-width="500px" class="docs-image--no-shadow" >}}

**Send now or schedule for later**

- **Send now** sends the report immediately after you save it. To stop sending the report at some point in the future, add an end date. If you leave the end date empty, the report is sent out indefinitely.

> - **Send later** schedules a report for a later date, so you must set a start date and time. If you leave the end date empty, the report is sent out indefinitely.

Reviewer 2: ```suggestion
- **Send later** schedules a report for a later date, if you set a start date and time. If you leave the end date empty, the report is sent out indefinitely.
```
The sentences cause an error in logic.

Author: I meant to say that because this options sends report later on a date, the date and time are required.

**Send only from Monday to Friday**

> You can choose to send some reports only from Monday to Friday. This applies to reports that have an hourly or daily frequency.

Reviewer 2: ```suggestion
For reports that have an hourly or daily frequency, you can choose to send them only from Monday to Friday.
```
